src/Compression-Tests/ZipArchiveTest.class.st

```diff
@@ -251,13 +251,13 @@ ZipArchiveTest >> testFilePermissions [
 ]
 
 { #category : #tests }
-ZipArchiveTest >> testParseSqueakZipTimestamp [
+ZipArchiveTest >> testParsePharoZipTimestamp [
 	"Issue 6054. We're testing a private method. Eventually, the zip timestamp handling should be refactored; maybe a ZipTimestamp subclass of DosTimestamp"
 
-	| squeakDosTimestamp dateAndTime |
+	| dosTimestamp dateAndTime |
 	"This was the error-causing timestamp from http://code.google.com/p/pharo/issues/detail?id=6054#c1 . It was the 'package' file inside Balloon-CamilloBruni.85.mcz"
```

Comment:
While at it, this link is dead. It would be good to update it, or explain the pitfalls.